### DISCUSSION TITLE
how to bypass securely with geogebra guide

### DISCUSSION THREAD

**Discussion by ilovetoucans** [3kh0/ext-remover#963] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
your going to need a ChromeOS running computer and geogebra installed in your apps, if your computer isnt for school this tutorial isnt going to be necessary as normal chrome computers dont automatically block websites. The first step is to just open geogebra, next you will want to go into the toolbar and click on the sign in button, next click on the forgot password button and go press the privacy button on that page, when you go on the privacy page there should then be another sign in page that displays sign in options which is where you click the more button, this page will display 3 choices and you will want to choose the microsoft one, you are now going to want to click on the microsofts sign in options where you then go choose the github one, when you get on the github page click the docs button, scroll on the docs main page until you see the still need help heading,then click the ask the communtity button and click on the tool bar in the top right corner and click on the search button and search up google and hit enter, a button should show up saying google.com when you look this up and now you should be able to bypass search blockers

**Comment by CGRmags** [comment]
@ilovetoucans this is public knowledge, You do know that?

**Reply by ilovetoucans** [reply]
I haven't seen a single person mention this online please shut the fuck up

**Reply by CGRmags** [reply]
@ilovetoucans damn language..... This is pretty much common knowledge, also this has been around for a while meaning schools have caught on. If they haven't then....Their tech department must be dumb.

**Reply by ilovetoucans** [reply]
my schools tech department really isn't that good at stopping new bypasses if they don't involve being a website besides GitHub as their only solution to stuff on Chromebooks that they don't like is just using securly

**Reply by CGRmags** [reply]
ok

**Reply by kelxit** [reply]
Google.com

**Reply by OrangeThunderFire** [reply]
Geogebra is blocked

**Reply by S-PScripts** [reply]
> @OrangeThunderFire ok sucks for u?

necropost moment

**Comment by OrangeThunderFire** [comment]
Yeah.

**Comment by PandaBoyMakesStuff** [comment]
Got patched
